liberica/src/lib/api.ts

```diff
@@ -1,18 +1,18 @@
 import axios from "axios";
-import { Team, Stop } from "lib/bindings";
+import { Team, Stop, CreateTeam } from "lib/bindings";
 
 const HOSTNAME_DEV = window.location.hostname;
 const HOSTNAME_PROD = window.location.host;
 
 export const BASE_URLS = import.meta.env.DEV
   ? {
-    WEBSOCKET: `ws://${HOSTNAME_DEV}:3000`,
-    FETCH: `http://${HOSTNAME_DEV}:3000/api`,
-  }
+      WEBSOCKET: `ws://${HOSTNAME_DEV}:3000`,
+      FETCH: `http://${HOSTNAME_DEV}:3000/api`,
+    }
   : {
-    WEBSOCKET: `wss://${HOSTNAME_PROD}`,
-    FETCH: `https://${HOSTNAME_PROD}/api`,
-  };
+      WEBSOCKET: `wss://${HOSTNAME_PROD}`,
```

Comment:
can we use wss / ws based on whether HTTP  or HTTPS is used? This would allow us to test the deployment locally